src/gpu/intel/jit/gemm/include/problem.hpp

```diff
@@ -178,11 +169,11 @@ struct GEMMProblem : public CommonProblem {
     bool postOpFwd = true;                          // Eltwise parameters
     bool cStochasticRound = false;
 
-    gpu_post_ops_t postOps;                         // Fused post operations to apply
-    std::bitset<post_ops_t::post_ops_limit> binaryRow;      // Binary op broadcasts row data if false
-    std::bitset<post_ops_t::post_ops_limit> binaryCol;      // Binary op broadcasts column data if false
-    std::bitset<post_ops_t::post_ops_limit> binaryBatch;    // Binary op broadcasts in batch dimension if false
-    std::bitset<post_ops_t::post_ops_limit> binaryTrans;    // Used to compute GEMMProblem::binary
+    PostOps postOps;         // Fused post operations to apply
+    std::bitset<maxPostOps> binaryRow;      // Binary op broadcasts row data if false
+    std::bitset<maxPostOps> binaryCol;      // Binary op broadcasts column data if false
+    std::bitset<maxPostOps> binaryBatch;    // Binary op broadcasts in batch dimension if false
+    std::bitset<maxPostOps> binaryTrans;    // Used to compute GEMMProblem::binary
```

Comment:
Can we make `maxPostOps` a static member of the `PostOps` type? I'd like to avoid global integer constants floating around...